src/components/CategoryFilter.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/prop-types */
+// import PropTypes from 'prop-types';
+// import { useSelector } from 'react-redux';
```

Comment:
- [ ] [OPTIONAL] It’s better to delete **commented piece of codes** like these. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))